app/config/templates/site.php

```diff
@@ -7,7 +7,7 @@
  */
 
 $protocol = System::getEnv('_APP_OPTIONS_FORCE_HTTPS') == 'disabled' ? 'http' : 'https';
-$hostname = System::getEnv('_APP_DOMAIN');
+$hostname = System::getEnv('_APP_CONSOLE_DOMAIN', System::getEnv('_APP_DOMAIN', ''));
```

Comment:
With this, let's set _APP_CONSOLE_DOMAIN=localhost in .env for local developemt, and tests.
Then, we can remove development override just below, and it should still work. Please test, by visitng Console and making sure templates have screenshots properly